src/CryptoAxus.Application/Dto/FixedPricedDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace CryptoAxus.Application.Dto;
+
+public class FixedPricedDto : BaseDto
+{
+    public string? NftCopyId { get; set; }
+
+    public int? Quantity { get; set; }
+
+    public decimal? PricePerItem { get; set; }
+
+    public DateTime? FixedEndingDateTime { get; set; }
+
+    public bool? SoftDelete { get; set; }
+
+    public FixedPricedDto()
+    {
+    }
+}
```

Comment:
Add another constructor that takes parameters for each property and assigns values to it.

Reply:
Resolved